app/controllers/publications/publication/decisions/index.js

```diff
@@ -27,13 +28,13 @@ export default class PublicationsPublicationDecisionsIndexController extends Con
     pieces.forEach((piece) => piece.publicationFlow = this.publicationFlow);
     yield Promise.all(pieces.map((piece) => piece.save()));
 
-    this.send('refresh');
+    this.router.refresh();
     this.closeReferenceDocumentUploadModal();
   }
 
   @dropTask
   *deletePiece(piece) {
     yield this.publicationService.deletePiece(piece);
-    this.send('refresh');
```

Comment:
both of these are too agressive.
Before we only reloaded the documents list on adding or deleting.
Now the parent route is refreshed.
using `this.router.refresh('publications.publication.decisions.index');` instead has the same effect as pre refactor.